mentat/code_map.py

```diff
@@ -0,0 +1,226 @@
+import json
+import subprocess
+import tempfile
+from pathlib import Path
+from typing import List
+
+from .llm_api import count_tokens
+
+
+def _get_git_files(git_root: str, absolute_paths: bool = False) -> List[str]:
+    git_files_ = (
+        subprocess.check_output(["git", "ls-files"], cwd=git_root)
+        .decode("utf-8")
+        .splitlines()
+    )
+    if absolute_paths:
+        git_files = [str(Path(git_root).joinpath(f)) for f in git_files_]
```

Comment:
We don't need 2 separate git_files variables; you can just overwrite git_files in the if statement and drop the else statement